runtime/vara/src/lib.rs

```diff
@@ -118,7 +118,7 @@ pub fn native_version() -> NativeVersion {
 
 /// Disallow balances transfer
 ///
-/// RELEASE: This is only relevant for the initial PoA run-in period and may be removed
+/// RELEASE: This is only relevant for the initial PoA run-in period and will be removed
 /// from the release runtime.
 #[derive(Default, Encode, Debug, Decode, Clone, Eq, PartialEq, TypeInfo)]
 pub struct DisableBalancesCall;
```

Comment:
Probably the struct itself should be renamed to reflect the new semantics, should it not?

Reply:
Into something like `DisableValueTransfers`?